orca_security/assets/logs/orca-security_tests.yaml

```diff
@@ -0,0 +1,240 @@
+id: "orca-security"
```

Comment:
The indentation for this file does not look correct. [You have an example here](https://github.com/DataDog/integrations-core/blob/83a9fe3553eee151579faf834897b554880c30e3/cisco_umbrella_dns/assets/logs/cisco-umbrella-dns_tests.yaml). Could you try make sure the message respect the same indentation. `Custom` should not be empty is done correctly

Reply:
For the log pipeline example, we usually receive the correct output with a custom tag from the validate-log action in the CI/CD pipeline, which we include in the results for all integrations. However, for Orca security, the custom tag appears empty, and despite this, the CI/CD pipeline still passes. We attempted to manually add values to the custom tag and adjusted the indentation according to the given reference. Unfortunately, this led to an internal server error with the message `An internal error occurred while assets were being validated.` and `Contact Datadog to find out how to resolve the problem.`

Reply:
`the correct output with a custom tag `
Indeed you should receive the correct response, but the sample need to properly formatted and I believe we struggle to make it work here. I believe how you committed here, [should be the good way](https://github.com/DataDog/integrations-core/blob/a3e51cd317c272b9af6fea1ee9bcb33b2d8c8607/orca_security/assets/logs/orca-security_tests.yaml). Can you try again and I will follow up to help

Reply:
Thanks. We have updated its format and added output with custom tag.